### DISCUSSION TITLE
PostgreSql 16. Doesn't show bases.

### DISCUSSION THREAD

**Discussion by andreysneg** [dbeaver/dbeaver#22145] [Q&A]
### Description

This is picture of dbeaver. Only 1 base.

![image](https://github.com/dbeaver/dbeaver/assets/14858838/213b5d3e-50a0-43f2-98b4-bffab5e4b83a)

This is picture of dbeaver settings:
https://bit.ly/3Nmh1Df

Buy i have 9 bases. For example from Navicat picture:
https://bit.ly/3REwa5q

Where is bases ? 

### DBeaver Version

Community 23.3.0

### Operating System

Windows 10

### Database and driver

Driver: PostgreSql 15
Database: PostgreSql 16.1

**Comment by moelbana** [comment]
Try to **GRANT CONNECT** access to one of them and see if it shows up in the navigator.

**Comment by andreysneg** [comment]
How ?

**Comment by andreysneg** [comment]
The settings were not the same as here https://bit.ly/3Nmh1Df

I set all checkboxex and i see all bases now. But i think all checkboxes must be setted by default. Every user want see all databases .

Thank you.

**Comment by moelbana** [comment]
I'm just gonna leave the answer here if anyone needs it, _I'm assuming you have a PostgreSQL username and password_:

You will need to grant this user **CONNECT** privilege on a specific database, open a command prompt, and navigate to the 

`PostgreSQL\version\bin` in your c: driver (don't forget to replace version with your installed version) and write this line:

> psql -U postgres -c "GRANT CONNECT ON DATABASE your_database TO your_username;"
 
replace `your_database` with the name of your database, and `your_username` with your PostgreSQL username.

Now your PostgreSQL username has a connect privilege to a specific database.

**Comment by LonwoLonwo** [comment]
Hello @andreysneg 

Please check your connection settings. Connection settings -> PostgreSQL tab -> `Show all databases`? Is it turned on?

![_1](https://github.com/dbeaver/dbeaver/assets/45152336/ff1077fa-9c93-4f7d-b487-6d0c9c04bafe)

**Comment by andreysneg** [comment]
Ок. Its worked
